core/src/main/java/jenkins/model/RssFeedItem.java

```diff
@@ -0,0 +1,50 @@
+package jenkins.model;
+
+import hudson.model.Run;
+import hudson.scm.ChangeLogSet;
+
+/**
+ * The type Rss feed item for Rss changelog.
+ */
+public class RssFeedItem {
```

Comment:
Why is this public? Best to put it back where it was. At a minimum

```suggestion
@Restricted(NoExternalUse.class)
public class RssFeedItem {
```